python/py-rpy2/Portfile

```diff
@@ -59,14 +59,22 @@ if {${name} ne ${subport}} {
                     port:py${python.version}-setuptools
 
     # To continue using your custom R installation instead of MacPorts' R,
-    # you have to remove this dependency line and make sure that your R
+    # you must remove this dependency line and make sure that your R
     # is in MacPorts' binpath, see your macports.conf for details.
     depends_lib-append  port:R
 
     # https://trac.macports.org/ticket/61327
-    if {![catch {set result [active_variants R openmp]}]} {
-        compiler.openmp_version 4.0
+    variant openmp description {enable parallelism support using OpenMP} {
+        # Please mirror the version used in math/R/Portfile
+        compiler.openmp_version 4.5
+
+        require_active_variants R openmp
+
+        notes-append "
+            Clang compilers provided by Xcode do not support OpenMP."
```

Comment:
these notes are not very useful IMHO - what's the purpose of this?

Reply:
This is a cut-and-paste from the `R` Portfile's `openmp` variant:

https://github.com/macports/macports-ports/blob/476eb5c0aeaf1d581a8266954ad260be3da3e576/math/R/Portfile#L134-L142

Reply:
there it makes more sense because of the second sentence; to me it's pretty useless in the `py-rpy2` port.

Reply:
Improved, thanks.